### ISSUE TITLE
Accessibility: governance intro

### ISSUE THREAD

**Issue by Lilianada** [jupyterhub/outreachy#78]
### What page is this for?

http://jupyter.org/governance/intro.html

### WAVE accessibility report

1. The <html lang> attribute is missing or empty, or a lang attribute value is not a valid language identifier.
Suggestion: Add the document language identifier using the attribute with a valid value (e.g., <html lang="en">).
2. Two or more <label>s are associated with a single <input>.
Suggestion: Change the "aria-label=" in the code to "aria-labelledby=".
3. Empty form label that does not contain any text or images with an alternative text, it will not present any information about the form control to the user.
Suggestion: A descriptive title attribute may be added to the form control instead of the label if it isn't visually necessary.
4. An element is present that contains no text content
Suggestion: Adding a value attribute to the button to state its value.

### The HTML element to be changed

```html
//ERRORS

//Language missing or invalid
<html style="margin-left: 380px;">

//Multiple form labels
<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-label="Toggle navigation sidebar">

<input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-label="Toggle in-page Table of Contents">

//Empty form label
<label for="__navigation" class="headerbtn" data-toggle="tooltip" data-placement="right" title="" data-original-title="Toggle navigation">
  <span class="headerbtn__icon-container">
     <i class="fas fa-list"></i>
   </span>
</label>

<label for="__page-toc" class="headerbtn headerbtn-page-toc">
  <span class="headerbtn__icon-container">
     <i class="fas fa-list"></i>
   </span>
</label>

<label for="toctree-checkbox-2">
  <i class="fas fa-chevron-down"></i>
</label>

//Empty Button
<button onclick="toggleFullScreen()" class="headerbtn" data-toggle="tooltip" data-placement="bottom" title="" data-original-title="Fullscreen mode">
   <span class="headerbtn__icon-container">
      <i class="fas fa-expand"></i>
   </span>
</button>

//ALERT
<a class="reference external" href="mailto:projectjupyter%40gmail.com">
projectjupyter
```

### Your proposed HTML (or CSS), after change

```html
//ERRORS

//Language missing or invalid
<html lang="en" style="margin-left: 380px;">

//Multiple form labels
<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-labelledby="Toggle navigation sidebar">

<input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-labelledby="Toggle in-page Table of Contents">

//Empty form labels
<label for="__navigation" class="headerbtn" data-toggle="tooltip" data-placement="right" title="" data-original-title="Toggle navigation">
<span class="" style="
    border: 0;
    padding: 0;
    margin: 0;
    position: absolute !important;
    height: 1px;
    width: 1px;
    overflow: hidden;
    clip: rect(1px 1px 1px 1px);
    clip-path: inset(50%);
    clip: rect(1px, 1px, 1px, 1px);
    white-space: nowrap;
">Menu bar</span>
<span class="headerbtn__icon-container">
  <i class="fas fa-bars"></i>
  </span>
</label>

or
<div class="headerbtn" data-toggle="tooltip" data-placement="right" title="" data-original-title="Toggle navigation">
  <span class="headerbtn__icon-container">
    <i class="fas fa-bars"></i>
  </span>
</div>


<label for="toctree-checkbox-2">
<span class="" style="
    border: 0;
    padding: 0;
    margin: 0;
    position: absolute !important;
    height: 1px;
    width: 1px;
    overflow: hidden;
    clip: rect(1px 1px 1px 1px);
    clip-path: inset(50%);
    clip: rect(1px, 1px, 1px, 1px);
    white-space: nowrap;
">Down icon</span>
   <i class="fas fa-chevron-down"></i>
</label>

or 

<div for="toctree-checkbox-2">
<i class="fas fa-chevron-down"></i>
</div>

<label for="__page-toc" class="headerbtn headerbtn-page-toc">
<span class="" style="
    border: 0;
    padding: 0;
    margin: 0;
    position: absolute !important;
    height: 1px;
    width: 1px;
    overflow: hidden;
    clip: rect(1px 1px 1px 1px);
    clip-path: inset(50%);
    clip: rect(1px, 1px, 1px, 1px);
    white-space: nowrap;
">
Menu bar
</span>
   <span class="headerbtn__icon-container">
     <i class="fas fa-list"></i>
   </span>
</label>

//Empty Button
<button onclick="toggleFullScreen()" class="headerbtn" data-toggle="tooltip" data-placement="bottom" title="" data-original-title="Fullscreen mode" value="button">
   <span class="headerbtn__icon-container">
      <i class="fas fa-expand"></i>
   </span>
</button>

//ALERT
<a class="reference external" href="#" >
projectjupyter
```

**Comment by minrk** [comment]
Thanks for the suggestions! The [aria-labelled-by](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby) attribute should contain an _id_ of another element containing the label text. It should not be the text itself.

I see you removed the mailto link from the email. Can you explain that in more detail?

**Comment by Lilianada** [comment]
My initial thought was to ignore the alert because the only way to not different buttons linked to a particular external link.

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
